sass/_header.scss

```diff
@@ -78,7 +78,7 @@
                 display: block;
 
                 input {
-                    padding: 0.2rem;
+                    border: 0.2rem;
```

Comment:
I changed that because on Mac in Safari is like that:

![image](https://user-images.githubusercontent.com/6381924/150596033-61f42b03-8484-4026-a586-f9bfdc1412cf.png)

And firefox (and Chrome) is like that:

![image](https://user-images.githubusercontent.com/6381924/150599198-83ba6cf9-394c-4fa4-9f0a-d5c70f06d494.png)

Reply:
@jenshaase I see what you mean, I revert the CSS tag.